frontend/src/app/services/reaction.service.ts

```diff
@@ -26,4 +28,46 @@ export class ReactionService {
     );
     return response;
   }
+
+  getArtistsByUserId(userId: number) {
+    const response = this.httpService.getRequest<ArtistReadDTO[]>(`/api/ArtistReaction/${userId}`);
+    return response;
+  }
+
+  addArtistReaction(artistId: number, userId: number) {
+    const response = this.httpService.postClearRequest(
+      '/api/ArtistReaction/like',
+      { artistId, userId }
+    );
+    return response;
+  }
+
+  removeArtistReaction(artistId: number, userId: number) {
+    const response = this.httpService.postClearRequest(
+      '/api/ArtistReaction/removeLike',
+      { artistId, userId }
+    );
+    return response;
+  }
+
+  getAlbumsByUserId(userId: number) {
+    const response = this.httpService.getRequest<AlbumForReadDTO[]>(`/api/AlbumReaction/${userId}`);
+    return response;
+  }
+
+  addAlbumReaction(albumId: number, userId: number) {
+    const response = this.httpService.postClearRequest(
+      '/api/AlbumReaction/like',
+      { albumId, userId }
+    );
+    return response;
+  }
+
+  removeAlbumReaction(albumId: number, userId: number) {
+    const response = this.httpService.postClearRequest(
+      '/api/AlbumReaction/removeLike',
+      { albumId, userId }
+    );
+    return response;
```

Comment:
I would say that you don't need to assign results to const, you could just return. Here and above in methods